source/geometries/FullRingInfinity.cc

```diff
@@ -296,17 +333,17 @@ void FullRingInfinity::BuildCryostat()
       new G4LogicalVolume(kapton_lat_solid, kapton, "KAPTON");
     G4double z_pos = axial_length_ / 2. + kapton_thickn_ / 2.;
     new G4PVPlacement(0, G4ThreeVector(0., 0., z_pos), kapton_lat_logic,
-                      "KAPTON_LAT_POS", LXe_logic_, false, 0, true);
+                      "KAPTON_LAT_POS", LXe_logic_, false, 0, false);
     new G4PVPlacement(0, G4ThreeVector(0., 0., -z_pos), kapton_lat_logic,
-                      "KAPTON_LAT_NEG", LXe_logic_, false, 1, true);
+                      "KAPTON_LAT_NEG", LXe_logic_, false, 1, false);
 
     // OPTICAL SURFACE FOR REFLECTION
     G4OpticalSurface *db_opsur = new G4OpticalSurface("BORDER");
     db_opsur->SetType(dielectric_metal);
     db_opsur->SetModel(unified);
     db_opsur->SetFinish(ground);
     db_opsur->SetSigmaAlpha(0.1);
-    db_opsur->SetMaterialPropertiesTable(petopticalprops::ReflectantSurface(0.));
+    db_opsur->SetMaterialPropertiesTable(petopticalprops::ReflectantSurface(wall_refl_));
```

Comment:
Will the inner faces be reflective now?

Reply:
Yes, to increase the light level on the SiPMs.